sdk/PrebidServerAdapter/PBServerAdapter.m

```diff
@@ -257,7 +257,10 @@ - (NSDictionary *)device {
 
 - (NSDictionary *)geo {
     CLLocation *clLocation = [[PBTargetingParams sharedInstance] location];
-    PBServerLocation *location = [PBServerLocation getLocationWithLatitude:clLocation.coordinate.latitude longitude:clLocation.coordinate.longitude timestamp:clLocation.timestamp horizontalAccuracy:clLocation.horizontalAccuracy];
+    PBServerLocation *location;
+    if (clLocation) {
+        location = [PBServerLocation getLocationWithLatitude:clLocation.coordinate.latitude longitude:clLocation.coordinate.longitude timestamp:clLocation.timestamp horizontalAccuracy:clLocation.horizontalAccuracy];
```

Comment:
What happens if clLocation does not exist, just don't initialize location?

Reply:
Yeah if the clLocation isn't set we don't send anything for location to Prebid Server.